engine/jobmaster/dm/task_manager.go

```diff
@@ -161,7 +167,7 @@ func (tm *TaskManager) checkAndOperateTasks(ctx context.Context, job *metadata.J
 }
 
 // remove all tasks, usually happened when delete jobs.
-func (tm *TaskManager) onJobNotExist(ctx context.Context) {
+func (tm *TaskManager) onJobDel(ctx context.Context) {
```

Comment:
Other cocurrent UpdateConfig from OpenAPI may overwrite Deleting

Reply:
update, check deleting in jobStore